examples/ElasticsearchDomain.py

```diff
@@ -36,7 +36,14 @@
                         'Action': 'es:*',
                         'Resource': '*'
                     }]},
-    AdvancedOptions={"rest.action.multi.allow_explicit_index": "true"}
+    AdvancedOptions={"rest.action.multi.allow_explicit_index": "true"},
+    VPCOptions={'SubnetIds': [
```

Comment:
This should be (and you'll need an import at the top for VPCOptions):
VPCOptions=VPCOptions(
    SubnetIds=["subnet-4f2bb123"],
    SecurityGroupIds=["sg-04cf048c"],
),

Reply:
Also, the "Ref" is a bit of a misnomer given the subnets/sg's aren't defined in this example. Really you're accessing already defined subnets/security groups.